### PR TITLE
H-1111: Update roadmap with new and progressed work

### PR DESCRIPTION
**Pull request by nonparibus**
## 🌟 What is the purpose of this PR?

- Updates the status of various nodes as they have progressed since last update.
- Makes explicit both in-flight and planned work not already well-represented on the roadmap.

This PR should not merge until any remaining _basic authorization_ PRs pending are in.

### CODE REVIEW

**Comment by nonparibus** [comment]
Ben's blocking PR has merged. @CiaranMn I believe you said there was one more to come? We don't have to rush this in tonight now, given how late information-site prep work went on, but let's aim to get it and this PR in tomorrow.

Marking as ready for review now _in case_ I had misunderstood -- and this is in fact now fully unblocked.

**Comment by nonparibus** [comment]
Follow-up: I see work is [in flight](https://github.com/hashintel/hash/commit/aa9e72ac80310569bb71d4dc687092c1b5c021a3). Have enabled auto-merge on this, so we should just approve when the blocking PRs (once opened) have been merged and tested.

**Comment by nonparibus** [comment]
Blocked by https://github.com/hashintel/hash/pull/3444

**Review by CiaranMn — APPROVED** [review]

**Pull request merged**
